make.paws/R/sdk_helper.R

```diff
@@ -1,67 +1,190 @@
 #' @import data.table
 #' @importFrom yaml write_yaml
-#' @importFrom devtools release
+#' @importFrom devtools release check check_rhub check_win_devel
+#' @importFrom urlchecker url_check
+#' @importFrom utils installed.packages remove.packages
 
 #' @title Check paws sdk
 #' @param in_dir Directory containing paws sdk packages.
 #' @param path Path to output paws sdk check results.
+#' @param pkg_list list of packages check locally, check all packages by default
 #' @param keep_notes Include note
 #' @return A list of any warnings, errors and notes within paws sdk.
+#' @name paws_check_local
 #' @export
```

Comment:
I don't think the `_cat` and `_sub_cat` functions need to be exported, but since this whole package is internal it doesn't really make a difference either way.